waspc/cli/src/Wasp/Cli/Interactive.hs

```diff
@@ -0,0 +1,109 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Wasp.Cli.Interactive
+  ( askForInput,
+    askToChoose,
+    askForRequiredInput,
+    Option,
+  )
+where
+
+import Control.Applicative ((<|>))
+import Data.Foldable (find)
+import Data.Function ((&))
+import Data.List (intercalate)
+import Data.List.NonEmpty (NonEmpty ((:|)))
+import qualified Data.List.NonEmpty as NE
+import qualified Data.Text as T
+import System.IO (hFlush, stdout)
+import Text.Read (readMaybe)
+import qualified Wasp.Util.Terminal as Term
+
+{-
+  Why are we doing this?
+
+  Using a list of Strings for options results in the Strings being wrapped in quotes
+  when printed. We want to avoid this so users can just type the name of the option
+  without having to type the quotes as well.
+
+  We do this by overriding the default Show instance for Strings with our own instance
+  that just returns the String itself.
```

Comment:
We don't override the default Show instance, we override the Option instance.

I would maybe emphasize that to solve the thing with Strings in quotes we introduced Option, which has default instance for every type implementing Show by just using Show, and only for String we do sometihng a bit different.